packages/patternfly-4/react-core/src/components/Radio/Radio.tsx

```diff
@@ -60,17 +62,20 @@ export class Radio extends React.Component<RadioProps> {
       <div className={css(styles.radio, className)}>
         <input
           {...props}
-          aria-label={label ? undefined : ariaLabel}
           className={css(styles.radioInput)}
           type="radio"
           onChange={this.handleChange}
           aria-invalid={!isValid}
           disabled={isDisabled}
           checked={checked || isChecked}
-          defaultChecked={isChecked ? undefined : defaultChecked}
+          {...!isChecked && { defaultChecked }}
```

Comment:
`defaultChecked` is an attribute passed to input? what does it do?

Reply:
I think it's required for the uncontrolled version of radio. The undefined was throwing a console error.